  Header header
  uint8 load_cell_id
  uint32 load_cell_grams
